src/linter/manifestJson/ManifestLinter.ts

```diff
@@ -57,6 +58,36 @@ export default class ManifestLinter {
 	}
 
 	#analyzeManifest(manifest: SAPJSONSchemaForWebApplicationManifestFile) {
+		if (manifest?.["sap.ui5"]?.dependencies?.minUI5Version) {
+			let availableVersions: string[] = [];
+
+			if (Array.isArray(manifest?.["sap.ui5"]?.dependencies?.minUI5Version)) {
+				availableVersions = manifest?.["sap.ui5"]?.dependencies?.minUI5Version;
+			} else if (typeof manifest?.["sap.ui5"]?.dependencies?.minUI5Version === "string") {
+				availableVersions.push(manifest?.["sap.ui5"]?.dependencies?.minUI5Version);
+			}
+
+			// Check if any version is below 1.136
+			const isBelow136 = availableVersions.some((version) => {
+				const normalizedVersion = semver.coerce(version);
+				return normalizedVersion && semver.lt(normalizedVersion, "1.136.0");
+			});
+
+			if (isBelow136) {
+				this.#reporter?.addMessage(MESSAGE.NO_LEGACY_UI5_VERSION_IN_MANIFEST,
+					"/sap.ui5/dependencies/minUI5Version");
+			}
+		}
+
+		if (manifest?._version?.startsWith("2.")) {
+			this.#validatePropertiesForManifestVersion(manifest, true);
+		} else {
+			this.#validatePropertiesForManifestVersion(manifest);
+			this.#reporter?.addMessage(MESSAGE.NO_OUTDATED_MANIFEST_VERSION, "/_version");
```

Comment:
Note that the manifest version 2 is currently not supported by the UI5 Language Assistant (see https://github.com/SAP/ui5-language-assistant/issues/748). We have to keep that in mind as people using the extension will see an error when changing the version to `2.0.0`.

Reply:
We had a meeting on July 7 where this was discussed with the owners of ui5-language-assistant and they agreed on an implementation plan, using `_version` and URLs like https://raw.githubusercontent.com/SAP/ui5-manifest/refs/tags/v2.0.0/schema.json to access the schema.

Reply:
Meanwhile, I got a reference to the corresponding action item on their side: 
Issue \#34962 in the (SAP internal) ux-engineering/tools-suite repo.